src/server/scripts/EasternKingdoms/zone_isle_of_queldanas.cpp

```diff
@@ -255,7 +255,7 @@ struct npc_thalorien_dawnseeker : public ScriptedAI
                     {
                         thalorien->SetWalk(true);
                         thalorien->GetMotionMaster()->MovePoint(0, thalorienFight);
-                        thalorien->SetUInt32Value(UNIT_NPC_EMOTESTATE, EMOTE_STATE_READY2H);
+                        thalorien->SetUInt32Value(UNIT_NPC_EMOTESTATE, 1717,);
```

Comment:
better return to enum.

Reply:
It was just [a mistake](https://github.com/TrinityCore/TrinityCore/pull/23382/commits/41d6972d3283cef8114b9a095ddacf4513b3486e) :rofl: